digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/CareClassTierekisteriImporter.scala

```diff
@@ -150,7 +150,7 @@ class CareClassTierekisteriImporter extends TierekisteriImporterOperations {
     sectionMeasures.foreach{ segment =>
       val trAssets = roadAddressInfo.filter {
         case (_, assetMeasures, _) =>
-          assetMeasures.startMeasure <= segment.startMeasure && assetMeasures.endMeasure >= segment.endMeasure && assetMeasures.endMeasure != assetMeasures.startMeasure//check values which are equal
+          assetMeasures.startMeasure - 0.001 <= segment.startMeasure && assetMeasures.endMeasure + 0.001 >= segment.endMeasure && assetMeasures.endMeasure != assetMeasures.startMeasure
```

Comment:
could be create a default val with tolerance info 

final private val DefaultEpsilon = 0.01